.github/workflows/test-node.yml

```diff
@@ -0,0 +1,54 @@
+name: Run Test
+
+on:
+  workflow_call:
+    inputs:
+      nim_wakunode_image:
+        required: true
+        type: string
+      test_type:
+        required: true
+        type: string
+
+env:
+  NODE_JS: "18"
+  RETRY_ATTEMPTS: 3
+  WAKU_SERVICE_NODE_PARAMS: ${{ (inputs.test_type == 'go-waku-master') && '--min-relay-peers-to-publish=0' || '' }}
+  DEBUG: ${{ (inputs.test_type == 'go-waku-master' || inputs.test_type == 'nwaku-master') && 'waku*' || '' }}
+
+jobs:
+  node:
```

Comment:
Github CI works in a way that if is sees a file in the `workflows` folder then it unfolds into steps in the CI. 

One way can be to re-use the action but then you need to place it into `actions` folder (similar to `actions/npm` I created there) and from ther you can use it

Or to create a file in `workflows` and expect it to be run in `js-waku` and `nwaku` and have conditions in it to handle that

Reply:
Tried to move it to `actions` but unfortunately it doesn't work. Actions is for jobs (like `npm`) not workflows and I need a workflow for the `workflow_call` functionality and cross repo usage.
So I need the new workflow to be excluded from the `Required status checks` in the Branch protection rules and made optional (similar to all other tests). 
@weboko do you have access to do this or do I need to ask @fryorcraken ? 
Thanks

Reply:
@weboko  @fryorcraken  little help here please

Reply:
@fbarbu15 was looking into it yesterday, to me it seemed that some different configuration might help here but if some setting on the repo required - I do not have access.

Reply:
Let me know if you have a better idea on how to handle this. 
On the other hand to fix it on my implementation we need something like this(tried it on a forked repo):
![image](https://github.com/waku-org/js-waku/assets/28725451/51947a18-96a8-43bc-8961-4d84c5b4a743)

Reply:
In that case we need to ask @fryorcraken to do the setting change 
+ I think we need to skip `node Expected — Waiting for status to be reported` in js-waku all together (seems it's already done)